src/VisualStudio/CSharp/Impl/ProjectSystemShim/CSharpProjectShim.cs

```diff
@@ -27,7 +27,7 @@ namespace Microsoft.VisualStudio.LanguageServices.CSharp.ProjectSystemShim
     /// effectively methods that just QI from one interface to another), are implemented here.
     /// </remarks>
     [ExcludeFromCodeCoverage]
-    internal abstract partial class CSharpProjectShim : CSharpProject
+    internal abstract partial class CSharpProjectShim : AbstractRoslynProject
```

Comment:
Eh, it's the shim. If anything else I'd also rename VB to "Shim" as well since this is really an adaptor between the two systems.